model/generate.go

```diff
@@ -764,8 +769,13 @@ func (g *GeneratedProject) addGeneratedProjectToConfig(intermediateProject *Pars
 						for j, intermediateProjectDT := range intermediateProjectBV.DisplayTasks {
 							if intermediateProjectDT.Name == dt.Name {
 								foundExisting = true
-								// avoid adding duplicates
-								_, execTasksToAdd := utility.StringSliceSymmetricDifference(intermediateProjectDT.ExecutionTasks, dt.ExecutionTasks)
```

Comment:
Previously, if we already had exec tasks {1,2,3} and generated {3,4,5}
this util function would have given us {1,2,4,5} to add to the already existing {1,2,3} which seems like a bug to me but someone should double check me on this